x-pack/plugins/cloud_security_posture/public/common/navigation/constants.ts

```diff
@@ -24,3 +24,8 @@ export const allNavigationItems: Record<CspPage, CspNavigationItem> = {
     disabled: !INTERNAL_FEATURE_FLAGS.showBenchmarks,
   },
 };
+
+export const findingsNavigation = {
+  findings_default: { name: TEXT.FINDINGS, path: '/findings/default' },
+  findings_by_resource: { name: TEXT.FINDINGS, path: '/findings/resource' },
+};
```

Comment:
these aren't in `allNavigationItems` because thats where `findings` is (as in `/findings`) 

we may need to think about this more, but for this PR it was easier than changing other things too. (like sidebar or component mapper)